test/integration/local_lambda/test_end_to_end.py

```diff
@@ -84,148 +70,170 @@ def invoke_function(self, port):
             f"http://localhost:{port}/2015-03-31/functions/function/invocations", json={}
         )
         
-    def create_container_and_invoke_function(self, cmd, port):
-        self.run_command(cmd)
-        
-        # sleep 1s to give enough time for the endpoint to be up to curl
-        self.sleep_1s()
-        
-        return self.invoke_function(port)
-
-    @parameterized.expand([("x86_64", "8000"), ("arm64", "9000"), ("", "9050")])
-    def test_env_var_with_equal_sign(self, arch, port):
+    @contextmanager
+    def create_container(self, cmd, image):
```

Comment:
This `cmd` should probably be called `params` instead, because it's not the full "command" anymore (same in each of the tests)

Reply:
sure